cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -100,10 +104,18 @@ func (cmd *command) run() error {
 
 	var err error
 	if cmd.K8s, err = kube.NewFromConfigWithTimeout("", cmd.KubeconfigPath, cmd.opts.Timeout); err != nil {
-		return errors.Wrap(err, "failed to initialize the Kubernetes client from given kubeconfig")
+		return fmt.Errorf("failed to initialize the Kubernetes client from given kubeconfig: %w", err)
+	}
+
+	if err := cmd.deploy(start); err != nil {
+		return err
+	}
+
+	if err := cmd.wizard(); err != nil {
```

Comment:
Wizard should not be enabled if `ci`-flag is set. At least we need a mechanism to turn off the default behaviour of opening the dashboard, i.e. by using the CI flag

Reply:
Absolutely, although I thought about it, I then did not add it here.